examples/.experimental/next-app-dir/src/trpc/rq-client.tsx

```diff
@@ -3,13 +3,17 @@
 import type { QueryClient } from '@tanstack/react-query';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
-import { loggerLink, unstable_httpBatchStreamLink } from '@trpc/client';
-import { createTRPCReact } from '@trpc/react-query';
+import {
+  createTRPCClient,
+  loggerLink,
+  unstable_httpBatchStreamLink,
+} from '@trpc/client';
+import { createTRPCContext } from '@trpc/tanstack-react-query';
 import type { AppRouter } from '~/server/routers/_app';
 import { useState } from 'react';
 import { createQueryClient, getUrl, transformer } from './shared';
 
-export const trpc = createTRPCReact<AppRouter>();
+export const { TRPCProvider, useTRPC } = createTRPCContext<AppRouter>();
```

Comment:
usually these are a tuple like


```suggestion
export const [ TRPCProvider, useTRPC ] = createTRPCContext<AppRouter>();
```

no big deal tho

Reply:
I'd probably favour named where it makes sense, like here. Means everyone adopts the same names for things by default, and that helps possible future codemods

Reply:
I don't think we have tuples anywhere else?